runtime/legion/legion_ops.inl

```diff
@@ -0,0 +1,38 @@
+/* Copyright 2018 Stanford University
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Included from legion_ops.h - do not include this directly
+
+
+namespace Legion {
+  namespace Internal {
+
+    //--------------------------------------------------------------------------
+    template<typename OP>
+    MemoizableOp<OP>::MemoizableOp(Runtime *rt)
+      : OP(rt)
+    //--------------------------------------------------------------------------
+    {
+    }
+
+    //--------------------------------------------------------------------------
+    template<typename OP>
+    void MemoizableOp<OP>::execute_dependence_analysis(void)
+    //--------------------------------------------------------------------------
+    {
+      OP::execute_dependence_analysis();
+    };
+  }; // namespace Internal
+}; // namespace Legion
```

Comment:
I feel like we don't actually need this file since all the use cases of the template are in the source file where it is also defined so they will automatically be instantiated in the same translation unit. Let's just put these in legion_ops.cc for now (or legion_ops.h if you want). I would like to keep '.inl' files to ones that are needed for user facing APIs only.